### PR TITLE
Adding Clearing Trash in Chaos Sanctuary & moving attack functions to classes

### PR DESCRIPTION
**Pull request by jobithu**
params.ini cs_clear_trash=1 now working.
To allow further classes to be built-in, diablo.py functions for attacking mobs are moved to the char.py for:
- all sorcs
- hammerdin
- trapsin
- necro (added lower resist & poison nova as skills to params.ini)

### CODE REVIEW

**Comment by jobithu** [comment]
still work in progress, no merge required for now.

**Comment by jobithu** [comment]
script working now, but still several pather got stuck -> more templates needed for the existing nodes 67x & 68x

File: src/char/i_char.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 class IChar:
-    def __init__(self, skill_hotkeys: Dict, screen: Screen, template_finder: TemplateFinder, ui_manager: UiManager):
+    def __init__(self, skill_hotkeys: Dict, char_config: Dict, screen: Screen, template_finder: TemplateFinder, ui_manager: UiManager):
```
Comment (aeon0) [inline src/char/i_char.py:19]: Dont add char_config back in please. It is taken for the config static class

**Review by aeon0 — CHANGES_REQUESTED** [review]
Overall this feels like lots of duplicate code :/ 
Also, please try to chop this PR up in smaller pieces. This is too much at once, impossible to review. E.g. 
1) Refactor code to a point that you can add the clear trash. (such as add pickit to ichar)
2) Implement it for Hdin
3) Separate PRs for other classes
4) Robustify pathing

**Comment by jobithu** [comment]
closing PR, will open multiple new ones as discussed & revert majority of pathing instructions back to diablo.py

**Pull request closed without merging**